src/main/java/CommandHandler.java

```diff
@@ -0,0 +1,71 @@
+import dukeexceptions.IllegalCommandException;
+import dukeexceptions.IllegalInputException;
+import items.Deadline;
+import items.Event;
+import items.Task;
+import items.ToDo;
+
+import java.time.LocalDate;
+
+public class CommandHandler {
+
+    private TaskList taskList;
+
+    public CommandHandler(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    public void execute(String command) throws IllegalCommandException,
+            IllegalInputException, NumberFormatException {
+        String[] parsedCommand = command.split(" ", 2);
+        switch (parsedCommand[0]) {
```

Comment:
I like your implementation! Perhaps you can consider further abstraction, e.g having an abstract Command class that different types of commands can extend from :)